src/os/layer/tile.js

```diff
@@ -295,7 +295,9 @@ os.layer.Tile.prototype.getBrightness = function() {
  */
 os.layer.Tile.prototype.getContrast = function() {
   if (this.layerOptions_) {
-    return /** @type {number} */ (this.layerOptions_['contrast'] || 1);
+    if (this.layerOptions_['contrast'] != null) {
```

Comment:
As minor cleanup, this if condition can be &&'d with the previous one. Same with the block in `getSaturation` below.